app/src/main/java/net/squanchy/service/firestore/FirestoreDbService.kt

```diff
@@ -141,9 +145,9 @@ class FirestoreDbService(private val db: FirebaseFirestore) {
 
     fun event(eventId: String): Observable<FirestoreEvent> {
         return Observable.create { subscriber ->
-            val registration = db.collection("views")
-                .document("event_details")
-                .collection("events")
+            val registration = db.collection(VIEWS)
+                .document(EVENT_DETAILS)
+                .collection(EVENTS)
                 .document(eventId)
                 .addSnapshotListener { snapshot, exception ->
                     if (exception != null && subscriber.isDisposed.not()) {
```

Comment:
No love for Tracks at line 166? 😢 

... I'll take care of it with the PR switching to using the tracks view, no need to do it now.

Reply:
got merged after I did it, but yeah we can do it